.eslintrc.js

```diff
@@ -37,8 +37,12 @@ module.exports = {
         'plugin:@typescript-eslint/recommended', // Uses the recommended rules from the @typescript-eslint/eslint-plugin
         'plugin:prettier/recommended', // Enables eslint-plugin-prettier and eslint-config-prettier. This will display prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
       ],
+      parserOptions: {
+        // This setting is required if you want to use rules which require type information
+        // https://github.com/typescript-eslint/typescript-eslint/tree/main/packages/parser#parseroptionsproject
+        project: ['./tsconfig.json', './tsconfig.test.json', './tsconfig.utils.json'],
```

Comment:
⚠️ We have specific `.eslintrc.js` files in the tests directories.

Reply:
I tried to move the parserOptions configuration in the dedicated eslintrc but I got errors.